src/dsp_tools/models/input_error.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+from typing import Protocol
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n\t"
+list_separator = "\n\t- "
+
+
+class Problem(Protocol):
+    """Information about input errors."""
+
+    def execute_error_protocol(self) -> str:
+        """
+        This function initiates all the steps for successful problem communication with the user.
+
+        Returns:
+            message for the error
+        """
+
+
+@dataclass(frozen=True)
+class DockerNotStartedProblem:
+    """This class contains information if the docker program is not started."""
+
+    user_msg: str
+
+
+@dataclass(frozen=True)
+class DirectoryProblem:
+    """This class contains information if a path does not exist."""
+
+    user_msg: str
+
+
+@dataclass(frozen=True)
+class FileNotFoundProblem:
+    """This class contains information if a file is not found at the expected location."""
+
+    user_msg: str
```

Comment:
I'm sure this is intentional - but these problem classes are missing the execute protocol method

Reply:
Same